test/services/nomisPushService.test.js

```diff
@@ -85,6 +85,31 @@ describe('nomisPushService', () => {
           data: { type: 'excluded', status: 'Yes', reason: 'deportation' },
           approvalStatus: { approvalStatus: 'INELIGIBLE', refusedReason: 'FNP' },
         },
+        {
+          example: 'Excluded - cja2003_19ZA',
+          data: { type: 'excluded', status: 'Yes', reason: 'cja2003_19ZA' },
+          approvalStatus: { approvalStatus: 'INELIGIBLE', refusedReason: 'CJA2003_19ZA' },
+        },
```

Comment:
This one doesn't seem to be listed anywhere else, has it been added previously and this is just adding it to a spec?

Reply:
Ah!, Forgot to add a comment for this sorry, it was missing from the test so added it in, it's one of the current reasons and just seemed weird that it was the only one left out of the test for ineligible reasons 😅